java/ql/lib/semmle/code/java/security/Encryption.qll

```diff
@@ -250,7 +250,7 @@ string getASecureAlgorithmName() {
   result =
     [
       "RSA", "SHA-?256", "SHA-?512", "CCM", "GCM", "AES(?![^a-zA-Z](ECB|CBC/PKCS[57]Padding))",
-      "Blowfish", "ECIES"
+      "Blowfish", "ECIES", "SHA3-(224|256|384|512)"
```

Comment:
```suggestion
      "Blowfish", "ECIES", "SHA3-(256|384|512)"
```

Let's drop the 224bits variant as it looks like SHA3-224 may be deprecated soon based on NIST's most recent [draft revision](https://csrc.nist.gov/pubs/sp/800/131/a/r3/ipd) of Transitioning the Use of Cryptographic Algorithms and Key Lengths